app/components/Siderbar/index.tsx

```diff
@@ -63,25 +59,17 @@ const MenuButton = styled(Button)`
   }
 `;
 
-const MenuImg = styled.img`
-  width: 1.7rem;
-  height: auto;
-  object-fit: contain;
-`;
-
 const Sidebar: React.FC = () => {
   const [visible, setVisible] = useState(false);
   const { isMobile } = useMedia();
 
   const toggleSidebar = useCallback(() => setVisible((v) => !v), []);
   const sidebarProps: DrawerProps = isMobile
     ? {
-        closeIcon: <CloseOutlined style={{ color: colors.secondary, fontSize: '1.9rem' }} />,
```

Comment:
What is mui replacement ?

Reply:
There is no such prop in mui, please refer latest commit for the changes.